chord_metadata_service/restapi/tests/test_argo.py

```diff
@@ -220,7 +220,7 @@ def setUp(self):
         self.primary_diagnosis = valid_cancer_condition()
 
     def test_get_argo(self):
-        create_object_response = get_response("cancercondition-list", self.primary_diagnosis)
+        create_object_response = get_response("cancerconditions-list", self.primary_diagnosis)
```

Comment:
btw I do not see anything called `cancerconditions-list`.  I see `cancerconditions` though.

Reply:
I don't know what it's supposed to be, but it seems to work now?